src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.cs

```diff
@@ -117,7 +117,21 @@ public static class Avx
         /// __m256d _mm256_cmp_pd (__m256d a, __m256d b, const int imm8)
         /// </summary>
         public static Vector256<double> Compare(Vector256<double> left, Vector256<double> right, FloatComparisonMode mode) => Compare(left, right, mode);
-        
+
+        /// <summary>
+        /// __m128d _mm_cmp_sd (__m128d a, __m128d b, const int imm8)
+        /// </summary>
+        public static Vector128<double> CompareScalar(Vector128<double> left, Vector128<double> right, FloatComparisonMode mode) => CompareScalar(left, right, mode);
+        /// <summary>
+        /// __m128 _mm_cmp_ss (__m128 a, __m128 b, const int imm8)
+        /// </summary>
+        public static Vector128<float> CompareScalar(Vector128<float> left, Vector128<float> right, FloatComparisonMode mode) => CompareScalar(left, right, mode);
+
+        /// <summary>
+        /// float _mm256_cvtss_f32 (__m256 a)
+        /// </summary>
+        public static float ConvertToSingle(Vector256<float> value) => ConvertToSingle(value);
```

Comment:
I see you provide the helper functions that convert vector to float/double. Do we need helpers for `float/double` -> `Vector128<float/double>`

Reply:
Do you mean providing `__m128 _mm_set_ss (float a)` in addition to `__m128 _mm_load_ss (float const* mem_addr)`?

Reply:
Yes, `SetScalar` sometimes can avoid memory access.

Reply:
👍, will add.

Reply:
Added `Vector128<float> Sse.SetScalar(float value)` and `Vector128<double> Sse2.SetScalar(double value)`